docs/getting-started/edfi-exchange/guides/edfi-6x-sandbox-mode-on-azure.md

```diff
@@ -28,33 +28,67 @@ Once the VM has been created, we need to configure IIS and add an SSL certificat
 * Remote Desktop (RDP) into the server.
 * Consider installing Notepad ++
 * Go to “Add features” for the server and enable Web Server (IIS).
-* Create a new c:\\inetpub\\Ed-Fi.
-* Copy "iisstart.htm" and "iisstart.png" from c:\\inetpub\\wwwroot to this new folder
+* Create a new `c:\inetpub\Ed-Fi`.
+* Copy "iisstart.htm" and "iisstart.png" from `c:\inetpub\wwwroot` to this new folder
 * Open IIS, create a new site “Ed-Fi.” Attach it to the new directory you just created.
 * Add a binding to the Ed-Fi site for port 80 (HTTP) using the DNS name that was just created.
 * Test that the server is externally accessible by putting the DNS name with http:// in a different machine’s browser. You should end up with the IIS landing page and a message about it being insecure.
-* Install an SSL certificate that is from a trusted provider. You can get one for free from Lets Encrypt using these instructions: [Free SSL Cert via Lets Encrypt](https://docs.google.com/document/d/1OBETP6lSVh35iCxZVYQhx9YeWXpOxk1NTqCCItvae0o/edit)
+* Install an SSL certificate that is from a trusted provider. You can get one for free from Lets Encrypt using these instructions:
```

Comment:
I didn't like seeing this link to a Google doc that is not in our control. Someone could change it to have something nasty. So I pasted the instructions below.